src/wazuh_db/wdb_parser.c

```diff
@@ -248,7 +248,7 @@ int wdb_parse(char * input, char * output, int peer) {
 
         // Don't perform this check if it's a manager.
         if (agent_id != 0) {
-            if (wdb_global = wdb_open_global(), !wdb_global) {
+            if (wdb_global = wdb_open_global(), !wdb_global || !wdb_global->enabled) {
```

Comment:
Do you think that we need a different message in case of `wdb_global->enabled == FALSE ` ? Maybe `Couldn't open DB global` sounds like an error, and in fact is a special situation.

Reply:
As we discussed with team, we shouldn't allow the whole upgrade process if the database isn't enabled.

Reply:
Fixed

Reply:
Fixed